nodeJS/.dockerignore

```
@@ -0,0 +1,2 @@
+node_modules
```

```suggestion
node_modules/
```
Pa mi que para que te ignore el directorio es necesario un "/"

---

No haría falta puesto que detecta que es un directorio y a partir del mismo no miraría nada más.

```
node_modules
```